cmd/manager/main.go

```diff
@@ -60,19 +62,21 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
-	var reclaimSpaceTimeout time.Duration
-	var maxConcurrentReconciles int
-	var enableAdmissionWebhooks bool
+	var (
+		metricsAddr             string
+		probeAddr               string
+		enableLeaderElection    bool
+		enableAdmissionWebhooks bool
+		cfg                     util.Config
+	)
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.DurationVar(&reclaimSpaceTimeout, "reclaim-space-timeout", defaultTimeout, "Timeout for reclaimspace operation")
-	flag.IntVar(&maxConcurrentReconciles, "max-concurrent-reconciles", 100, "Maximum number of concurrent reconciles")
+	flag.DurationVar(&cfg.ReclaimSpaceTimeout, "reclaim-space-timeout", defaultTimeout, "Timeout for reclaimspace operation")
```

Comment:
It is cleaner to have default values in `cfg` set by default. Can you introduce `util.NewConfig()` and move the defaults to `internal/utils/config.go`? It'll make the unit test a little easier too, no need to pass the defaults for every test.